app/javascript/components/upload/upload-utils.js

```diff
@@ -249,8 +249,8 @@ const plainTextExtensions = ['.txt', '.tsv', '.text', '.csv']
 const mtxExtensions = ['.mtx', '.mm', '.txt', '.text']
 const imageExtensions = ['.jpeg', '.jpg', '.png', '.bmp']
 const miscExtensions = ['.txt', '.text', '.tsv', '.csv', '.jpg', '.jpeg', '.png', '.pdf',
-  '.doc', '.docx', '.xls', '.xlsx', '.ppt', '.pptx', '.zip', '.loom', '.h5', '.h5ad', '.h5an',
-  '.ipynb', '.Rda', '.rda', '.Rds', '.rds']
+  '.doc', '.docx', '.xls', '.xlsx', '.ppt', '.pptx', '.zip', '.loom', '.h5an',
```

Comment:
I think we can safely remove "h5an", as I can find no relevant reference to this file extension.

Its existence in this list derives from [a request on 2020-03-10 in our #scp Slack channel](https://broadinstitute.slack.com/archives/C9YSEMSFR/p1583863287031200), and searching Slack for "h5an" returns [one prior hit in #scp-implementation from 2019-03-15](https://broadinstitute.slack.com/archives/CBEHTH601/p1552673056000300) with a link to https://scanpy.readthedocs.io/en/latest/api/scanpy.read_10x_h5.html.  That link no longer mentions that extension, and the Scanpy repo doesn't mention it, so at most I suspect it was an equivalent alternative to .h5 or .h5ad that briefly existed and is now no longer used at all.

```suggestion
  '.doc', '.docx', '.xls', '.xlsx', '.ppt', '.pptx', '.zip', '.loom',
```